bin/list_schemes.sh

```diff
@@ -2,7 +2,32 @@
 
 set -o pipefail
 
-xcrun xcodebuild -list "$@" 2>/dev/null \
+while getopts "f:t:i:" opt; do
+  case $opt in
+    f) flag="$OPTARG";;
```

Comment:
Nit: is there a more descriptive name here other than `flag`? Maybe `target_type_flag`? Not super important, I can just see myself being confused by this at some point in the future.

Reply:
Done here!: ded7e553708d5d4bcfe56a50c95efc17bb40cc6b